SWEgit/src/com/company/Main.java

```diff
@@ -198,4 +198,25 @@ public static void GetAverage(String[] arr) {
         System.out.println("Average : "+sum/arr.length);
         }
     }
+    public static void getmin3nums(int[]arr)
```

Comment:
The input will be string array in the main so i suggest to edit your function to be string array and covert this array to integer